### DISCUSSION TITLE
Auto-formatting of .60 files

### DISCUSSION THREAD

**Discussion by Zajozor** [slint-ui/slint#895] [Q&A] [answered]
Hello, thanks for the good work! Awesome framework!

Is there currently any way to achieve the "auto format on save" behavior eg in Vscode? I couldn't really find any reference about it.
Or is it built into the plug-in and I just have it misconfigured?

It gets a little cumbersome to manually do all the tabbing, etc.

If it's not possible right now, is it planned? How/where is the help needed to implement it ?

Thanks!

**Comment by ogoffart** [comment, marked as answer]
Hi, 
Thanks for your interest!
We've started developing a formatting tool. It's there:  https://github.com/sixtyfpsui/sixtyfps/blob/master/tools/fmt/main.rs

But it's only very basic now, just some work in progress.

Eventually we will get a formatter, that's the plan.
